webpack.demo.js

```diff
@@ -0,0 +1,54 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'angular2': [
+      'zone.js',
+      'reflect-metadata',
```

Comment:
There is an issue in here. I was going to do it as Pawel said, but that would require the final demo to have a node_modules with the dependencies or something. I mean if I put script tags like:

```
<script src="/node_modules/angular2/node_modules/reflect-metadata/Reflect.js"></script>
<script src="/node_modules/angular2/node_modules/zone.js/dist/zone.js"></script>
```

I need that path to exist on gh-pages. If I use the `external-dependencies` thingy, I would need also to have a node_modules with angular in it for the gh-pages.

The idea is to serve just the `index.html` and 1-2 bundle files with minified dependencies. Not a node_modules with stuff in it.

Perhaps I am doing it wrong.